check-format-cpp.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Apache License 2.0
+
+project_directory="$(git rev-parse --show-toplevel)"
+
+pushd "${project_directory}" > /dev/null
```

Shellcheck (0.10.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)